tsc/CMakeLists.txt

```diff
@@ -49,7 +52,12 @@ option(ENABLE_MRUBY "Enable the MRuby scripting engine" ON)
 option(ENABLE_NLS "Enable translations and localisations" ON)
 option(ENABLE_EDITOR "Enable the in-game editor" ON)
 option(USE_SYSTEM_TINYCLIPBOARD "Use the system's tinyclipboard library" OFF)
-option(USE_SYSTEM_MRUBY "Use the system's mruby (intended for internal use only!)" OFF)
+if(MSYS)
+  option(USE_SYSTEM_MRUBY "Use the system's mruby (intended for internal use only!)" ON)
+  set(CMAKE_INSTALL_PREFIX "C:/Games/TSC" CACHE PATH "CMAKE PREFIX" FORCE)
```

Comment:
This looks like a fixed path that only works on your system. Anyone with a different path setup will likely experience trouble. Why do you need this path? If needed, can it be extracted into a variable?

Reply:
I have previously built TSC on msys2 with the built-in mRuby. But, as of now, the build in one complains during CMake Configure about prefixes being different `/c/something` vs `C:/something`, which msys2 handles just fine.

Reply:
CMake on Windows defaults install prefix to `C:/Program Files (x86)/${CMAKE_PROJECT_NAME}`, I generally avoid installing to PF. I can replace this with `$ENV{systemdrive}/Games/${CMAKE_PROJECT_NAME}` if that's better.

Reply:
I agree with you for customly compiled software. I wouldn't want that in `C:\Program Files` either. However, if installed via the installer, it should go there, because TSC is not different from other programmes then. Could your code be amended such that

1. `CMAKE_INSTALL_PREFIX` defaults to `C:/Games/TSC`, but can be overridden on the commandline, and
2. The installer will install into `C:\Program Files` without the need to pass any option

? If you don't want, it's fine, then I'll take a look it at that after merging. It's been a while since I last fiddled with cpack, otherwise I'd give you a suggestion here.

Reply:
Now `CMAKE_INSTALL_PREFIX` will be set to `$ENV{systemdrive}/Games/${CMAKE_PROJECT_NAME}` only when the user did not specify one in the arguments.
CPackdoes not inherit that value, so nothing to change there.